src/olympia/lib/settings_base.py

```diff
@@ -1289,7 +1289,23 @@ def read_only_mode(env):
 STATIC_ROOT = path('site-static')
 STATIC_URL = '/static/'
 
-STATICFILES_DIRS = (path('static'),)
+STATICFILES_FINDERS = (
+    'django.contrib.staticfiles.finders.FileSystemFinder',
+    'django.contrib.staticfiles.finders.AppDirectoriesFinder',
+    'django_node_assets.finders.NodeModulesFinder',
+)
+
+NODE_MODULES_ROOT = os.path.join('/', 'deps', 'node_modules')
+NODE_PACKAGE_JSON = os.path.join('/', 'deps', 'package.json')
+NODE_PACKAGE_MANAGER_INSTALL_OPTIONS = ['--dry-run']
+
+STATIC_DIST_PATH = os.path.join('/', 'data', 'olympia', 'static-dist')
```

Comment:
As discussed on slack, let's rename the constant and the directory to `static-build` (and `STATIC_BUILD_PATH`) and document what it is for.